Framework/Core/include/Framework/SimpleRawDeviceService.h

```diff
@@ -37,8 +38,19 @@ class SimpleRawDeviceService : public RawDeviceService
     mDevice = device;
   }
 
+  DeviceSpec const& spec() final
+  {
+    return mSpec;
+  }
+
+  void setSpec(DeviceSpec const& spec) final
+  {
+    mSpec = spec;
+  }
+
  private:
   FairMQDevice* mDevice;
+  DeviceSpec mSpec;
```

Comment:
Why a copy?

Reply:
indeed, its a ref now which is set in the constructor